packages/devtools_app/lib/src/shared/config_specific/server/_server_web.dart

```diff
@@ -420,6 +421,70 @@ Future<List<String>> requestAndroidBuildVariants(String path) async {
   return const <String>[];
 }
 
+Future<AppLinkSettings> requestAndroidAppLinkSettings(
+  String path, {
+  required String buildVariant,
+}) async {
+  if (isDevToolsServerAvailable) {
+    final uri = Uri(
+      path: DeeplinkApi.androidAppLinkSettings,
+      queryParameters: {
+        DeeplinkApi.deeplinkRootPathPropertyName: path,
+        DeeplinkApi.androidBuildVariantPropertyName: buildVariant,
+      },
+    );
+    final resp = await request(uri.toString());
+    if (resp?.status == HttpStatus.ok) {
+      return AppLinkSettings.fromJson(resp!.responseText!);
+    } else {
+      logWarning(resp, apiResetDevTools);
```

Comment:
this should be DeeplinkApi.androidAppLinkSettings instead of apiResetDevTools. Similar for updating the logWarning calls below